### DISCUSSION TITLE
Remix rate limiting issue - client script bundling ?

### DISCUSSION THREAD

**Discussion by rreeves8** [remix-run/remix#8925] [Q&A] [answered]
Hi

We are using remix on aks, and we have implemented rate limiting which is causing an issue with the number of script requests remix makes. I checked the network tab and the number of requests seems pretty high. 

<img width="241" alt="Screenshot 2024-02-28 at 9 56 51 AM" src="https://github.com/remix-run/remix/assets/71095829/5d68271f-5f0f-41e1-b63a-1f00fe284d9c">

We are also pre fetching 1 other route on page load. However I was wondering if there is a way to bundle the scripts per route ? to avoid making many requests per page.

Or if anyone has any other solutions. 

We have limited it to 35 requests at a time, which is suitable for our application. 

Thanks

**Comment by sergiodxa** [comment, marked as answer]
There's no way to configure the number of chunks that are generated, I recommend you to instead put a CDN in front of your app, then your static assets can be requested from the CDN instead of your app so they will not count for the 35 request limit you set.

Another option is that you could grab the public folder and upload it to S3 and send the traffic there, then configure in remix.config.js to use another domain for those assets, check https://sergiodxa.com/tutorials/use-a-cdn-for-your-static-assets-in-remix.
